amun/overlays/test/imagestreamtag.yaml

```diff
@@ -26,3 +26,17 @@ spec:
       importPolicy: {}
       referencePolicy:
         type: Source
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: kubectl
+spec:
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: quay.io/thoth-station/kubectl:latest
```

Comment:
do we have an "official" kubernetes or OpenShift image for this?

Reply:
we have openshift image but this also containes jq. 
Here: `quay.io/openshift/origin-cli:latest`